flake8_picky_parentheses/_redundant_parentheses.py

```diff
@@ -229,7 +228,8 @@ def separate_logic_lines(source_code, start_tree, all_logic_lines):
         for counter in range(logic_line[0], logic_line[1]):
             code_to_check.append(source_code[counter])
         for line_num in range(len(code_to_check)):
-            if line_num < checked_lines or code_to_check[line_num] == "":
+            if (line_num < checked_lines or code_to_check[line_num] == ""
+               or code_to_check[line_num].isspace()):
```

Comment:
Firstly, I don't like the looks of this indentation.
Go with
```python
            if (
                line_num < checked_lines or code_to_check[line_num] == ""
                or code_to_check[line_num].isspace()
            ):
```

or 

```python
            if (line_num < checked_lines or code_to_check[line_num] == ""
                    or code_to_check[line_num].isspace()):
```

Secondly, this condition can be simplified:
```suggestion
            if (line_num < checked_lines
                    or code_to_check[line_num].trim() == ""):
                continue
```